.gitignore

```diff
@@ -16,7 +16,6 @@ Thumbs.db
 
 # Wordpress
 wp-content/*
-!wp-content/themes/
```

Comment:
Oh, whoops! I added a test `wp-content` directory at the top level of the repo for testing plugins and themes and whitelisting the themes directory made the test theme that I was using visible to this git repo.

Do you think we should add another gitignore file in the example directory?